spec/dummy/config/initializers/react_on_rails.rb

```diff
@@ -0,0 +1,3 @@
+ReactOnRails.configure do |config|
+  config.bundle_js_file = "app/assets/javascripts/generated/server.js"
```

Comment:
@alexfedoseev this is a good point. I think we can do the following:

1. `bundle_js_file` is the default
2. the bundle file can be overridden like the prerender option
3. we offer a setup for multiple bundles after v.1.0 which would have the ability to setup a common file, and then the calls to react_component would specify the option of the `bundle_js_file`, and this would be combined with the common

### HOWEVER
If we're caching the server.js file on the Rails server between requests, why not just have **ONE** server.js file that can render any required component.

I'm not seeing what could be the advantage of separate bundles for the server side.

I can see an advantage for the client side in terms of not sending extra JS. What's the advantage on the server side?

CC: @samnang